src/app/profile/my-spaces/my-spaces-search-dialog/my-spaces-search-spaces-dialog.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="modal fade" bsModal #modal="bs-modal" tabindex="-1" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title pull-left">Find Spaces to Join</h4>
+        <button type="button" class="close pull-right" aria-label="Close" (click)="hide()">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+
```

Comment:
Some of the new lines look strange to me; I think I'd prefer not having them. Mainly the ones that are followed by an indented element. Ones followed by an element at the same indent level are okay.